src/main/java/input/file/PlainTextFile.java

```diff
@@ -0,0 +1,50 @@
+package input.file;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+/**
+ * Class to specify a .txt file.
+ * @author Paul
+ *
+ */
+public class PlainTextFile extends DataFile {
+	
+	public PlainTextFile(String path) {
+		super(path);
+	}
+	
+	@Override
+	public InputStream getDataStream() throws FileNotFoundException {
+		InputStream stream = new FileInputStream(getFile());
+		Scanner scanner = new Scanner(stream, "UTF-8");
+		scanner.useDelimiter("\\A");
+		StringBuilder builder = new StringBuilder();
+		int i = 1;
+
+		//Skip to beginline
+		while (i != getStartLine() && scanner.hasNextLine()) {
```

Comment:
Het exception throwen is bewust niet gedaan. Dit was als er geen start en/of endline is gespecificeerd door de user. dan moet gewoon tot het eind van de file worden gelezen